packages/flutter/lib/src/material/chip.dart

```diff
@@ -2425,6 +2424,31 @@ bool _hitIsOnDeleteIcon({
   };
 }
 
+class _DeleteButtonSemantic extends SingleChildRenderObjectWidget {
+  const _DeleteButtonSemantic({super.child});
+
+  @override
+  RenderObject createRenderObject(BuildContext context) {
+    return _RenderDeleteButtonSemantic();
+  }
+}
+
+class _RenderDeleteButtonSemantic extends RenderProxyBox {
+  @override
+  void describeSemanticsConfiguration(SemanticsConfiguration config) {
+    super.describeSemanticsConfiguration(config);
+    config.isSemanticBoundary = true;
+    config.isButton = true;
+  }
+
+  @override
+  Rect get semanticBounds => Rect.fromCenter(
+    center: paintBounds.center,
+    width: kMinInteractiveDimension,
+    height: kMinInteractiveDimension,
```

Comment:
It is not always this though, based on the MaterialTapTargetSize and VisualDensity, it could be a different rect correct?

Reply:
I hesitated on this because it is not clear from the Material spec that the tap target size should change based on visual density.
But you're right: it is preferable to follow what is done in most of the Flutter Material widget.

I updated he PR, using the same logic than several Flutter widgets (radio, checkbox). Which means tap target size defaults to:
- 48x48 on mobile
- 32x32 on desktop

For reference, see https://github.com/flutter/flutter/blob/c02d22d8ad39cdf4e0a16750784e8d475c9f038e/packages/flutter/lib/src/material/checkbox.dart#L519-L529

Reply:
actually nvm, there is a clause in WCAG as well
https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum
and the lower bound is also 24. we should be fine here